glommio/src/timer/timer_impl.rs

```diff
@@ -1127,11 +1132,10 @@ mod test {
                     }
                 }
             });
-            Timer::new(Duration::from_millis(100)).await;
```

Comment:
removing this line and not replacing by anything is not great.
I understand the timing issue, but the goal of that line was to make sure that we're counting to 10 over time, through the repeat action and not, for example, that the timers are firing all at once.

So we should at least capture a timestamp before and after and make sure that enough time has passed.

Reply:
Good point. I added an assertion about elapsed time.

Reply:
Hmmm, the test failed because of that in the github test (passed on my local VM). I need to look more into it.

Reply:
The failed test is fixed (the minimum time should be 45ms instead of 50).

The nightly doc is still failing but should not be a blocker.

 Please take another look. Thanks!

Reply:
@HippoBaro any idea what's going on with the doctest ?

Reply:
Rust nightly is panicking. We should submit bug report to the rust folks

Reply:
Thank you! I filed https://github.com/rust-lang/rust/issues/95793 for the rust nightly bug.